namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-dev/resources/s3-metrics.tf

```diff
@@ -32,3 +32,67 @@ resource "kubernetes_secret" "book_a_secure_move_metrics_s3_bucket" {
     bucket_name       = module.book_a_secure_move_metrics_s3_bucket.bucket_name
   }
 }
+
+# Generate an additional IAM user with read-only access to the bucket
+resource "random_id" "id" {
```

Comment:
it seems there are failures in concourse around duplicates in resource names found here: https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-dev/resources/s3-reporting.tf#L38

Reply:
👍